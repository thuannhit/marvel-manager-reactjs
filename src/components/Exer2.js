import React, {Component} from 'react';
import Paper from '@material-ui/core/Paper';
import Typography from '@material-ui/core/Typography';
class Ex2 extends Component{
    
    render(){
        return (
            <div>
                <br/>
                <Paper elevation={1}>
                    <Typography variant="h5" component="h3">
                        Using your knowledge to make the code better
                    </Typography>
                    <Typography component="p">
                        Please see the .docx file attached
                        <a href='https://1drv.ms/w/s!AsbPIUcZlBUXwEU9dfl3f0ATd9xL' target="_blank" download> Click to open</a>
                    </Typography>
                </Paper>
            </div>
        )
    }
}

export default Ex2;