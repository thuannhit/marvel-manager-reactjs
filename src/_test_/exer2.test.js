import React from 'react';
import Exer2 from '../components/Exer2';
import renderer from 'react-test-renderer';

it('Exer2 renders correctly', () => {
  const tree = renderer
    .create(<Exer2/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
