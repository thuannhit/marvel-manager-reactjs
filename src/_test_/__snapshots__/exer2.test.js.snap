// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exer2 renders correctly 1`] = `
<div>
  <br />
  <div
    className="MuiPaper-root-1 MuiPaper-elevation1-4 MuiPaper-rounded-2"
  >
    <h3
      className="MuiTypography-root-28 MuiTypography-h5-44"
    >
      Using your knowledge to make the code better
    </h3>
    <p
      className="MuiTypography-root-28 MuiTypography-body1-37"
    >
      Please see the .docx file attached
      <a
        download={true}
        href="https://1drv.ms/w/s!AsbPIUcZlBUXwEU9dfl3f0ATd9xL"
        target="_blank"
      >
         Click to open
      </a>
    </p>
  </div>
</div>
`;
